spec/index.bs

```diff
@@ -153,22 +153,58 @@ div.content-timeline, div.device-timeline, div.queue-timeline {
     background: var(--tint-purple);
 }
 
+[data-timeline] {
+    border-style: solid;
+    border-width: thin;
+    border-radius: .2rem;
+}
+span[data-timeline] {
+    border-width: 0 0 thin 0;
+}
+div[data-timeline] {
+    border-width: 1px 0 0 .75em;
+    border-radius: 1em 0 0 1em;
+    padding: 1.5em .5em .5em .5em;
+}
+div.algorithm > div[data-timeline] {
+    /* Remove unnecessary padding in this common case. Makes the border of the
+     * timeline box line up with the border of the surrounding algorithm box. */
+    margin-left: calc(-.5em - 1px);
+    margin-right: calc(-.5em - 1px);
+}
+
+/*
+ * Coloring for steps and other stuff on particular timelines.
+ */
+[data-timeline="content"] {
+    border-color: #00cc00 !important;
```

Comment:
The content/device timeline colors are much easier for me to parse in this version, thanks!